docs/api.md

```diff
@@ -343,7 +343,14 @@ discussion of the advanced features exposed in the Command API.
 Enable bash/zsh-completion shortcuts for commands and options.
 
 `cmd`: When present in `argv._`, will result in the `.bashrc` or `.zshrc` completion script
-being outputted. To enable bash/zsh completions, concat the generated script to your
+being outputted.
+
+To enable bash/zsh completions, you can either: 
+1. Write it to a file in `$XDG_DATA_HOME/bash-completion` or
```

Comment:
This bash description will only work if the `bash-completion` package is installed. I checked macOS and Debian and Ubuntu and Fedora and none of them had it installed by default.

Sadly appending to the shell start-up scripts is the lowest common denominator! I suggest have that first, and then the zsh autoload instructions.

Perhaps rather than using filename placeholder here, could point to the completion text where it can be included. Like:

> 2. For Zsh autoloading, write it to a to a file in your `$FPATH`. See completion script for required file name.